_build_cfg.yml

```diff
@@ -180,6 +180,8 @@ Topics:
     File: securing_builds
   - Name: Building Dependency Trees
     File: building_dependency_trees
+  - Name: Web Console Extensions
```

Comment:
would "Customization" instead of "Extensions" make it easier to find this topic?